### PR TITLE
added polygon functions

### PR DESCRIPTION
**Pull request by radamejr**

### CODE REVIEW

File: 37-add-polygon/demo.details [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: AerialSphere addPolygon
+description: Adds a polygon to the viewer
+authors:
+    - AerialSphere
+resources:
+    - https://app.aerialsphere.com/api/map/js?key=4f0e9801-ab77-4394-874e-d56404ab17db
```
Comment (bobsbeenjamin) [inline 37-add-polygon/demo.details:7]: Can we get away without committing this?

File: 37-add-polygon/demo.details [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+name: AerialSphere addPolygon
+description: Adds a polygon to the viewer
+authors:
+    - AerialSphere
```
Comment (bobsbeenjamin) [inline 37-add-polygon/demo.details:5]: Might be nice to add your name as an author

File: 38-set-polygon-distance/demo.details
Comment (bobsbeenjamin) [inline 38-set-polygon-distance/demo.details:1]: Same comments here as `37-add-polygon/demo.details`

**Review by bobsbeenjamin — CHANGES_REQUESTED** [review]

File: 37-add-polygon/demo.details (hunk 1 above)
Comment (radamejr) [reply 37-add-polygon/demo.details:7]: the resources? no
That lets it pull the javascript api...

**Review by bobsbeenjamin — APPROVED** [review]
LGTM

**Pull request merged**